doc-theme/theme.scss

```diff
@@ -17,21 +16,32 @@ $posit-green:   #72994E;
 $posit-burgundy:#9A4665;
 $posit-burgundy-1:#78384F;
 $posit-burgundy-2:#542938;
+
+$primary: $posit-blue;
 // scss-docs-end color-variables
 
 // Typography
 // Font, line-height, and color for body text, headings, and more.
 
 //scss-font-start font-variables
-$font-family-sans-serif: "Open Sans",   system-ui, -apple-system, "Segoe UI", Roboto, "Helvetica Neue", "Noto Sans", "Liberation Sans", Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji" !default;
-$font-size-base: 1rem !default;
+$font-family-sans-serif: "Open Sans", "Segoe UI", Roboto, "Helvetica Neue", "Noto Sans", "Liberation Sans", Arial, sans-serif, system-ui;
```

Comment:
Broke this up because the "emoji" or "symbol" fonts are...well, emoji. Sans serif is going to be our main text. You probably don't need quite this many options, a person's browser is going to stop as soon as it finds one it recognizes -- that's also why I put `system-ui` at the end, since that's pretty much the ultimate fallback. 

You could probably get away with Open Sans, Helvetica, Arial, sans-serif -- if their browser doesn't recognize any of those, what on earth are they using? a command line?

Reply:
this makes sense to me